charts/seed-monitoring/charts/prometheus/rules/kube-etcd3.rules.yml

```diff
@@ -2,19 +2,30 @@ groups:
 - name: kube-etcd3.rules
   rules:
 
-  # alert if another failed peer will result in an unavailable cluster
-  - alert: KubeEtcd3InsufficientPeers
-    expr: absent(up{job="kube-etcd3",role="main"})
-      or absent(up{job="kube-etcd3",role="events"})
-      or count(up{job="kube-etcd3"} == 0) by(role) > (count(up{job="kube-etcd3"}) by (role) / 2 - 1)
+  # alert if etcd is down
+  - alert: KubeEtcd3Down
+    expr: absent(up{job="kube-etcd3",role="main"}) or absent(up{job="kube-etcd3",role="events"})
     for: 3m
     labels:
       service: etcd
       severity: critical
       type: seed
     annotations:
-      description: If one more etcd3 peer goes down the etc3 cluster {{ $labels.role }} will be unavailable
-      summary: etcd3 cluster small
+      description: If etc3 cluster {{ $labels.role }} is be unavailable or cannot be scrapped
+      summary: etcd3 cluster down
+
+  # TODO: enable this alert for HA scenarios
```

Comment:
Not sure why we keep that whole section; there are currently no plans to do anything regarding HA?